### PR TITLE
adds bulk secret

### PR DESCRIPTION
**Pull request by mchmarny**
Implements the latest protos

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -57,6 +57,9 @@ type Client interface {
 	// GetSecret retreaves preconfigred secret from specified store using key.
 	GetSecret(ctx context.Context, storeName, key string, meta map[string]string) (data map[string]string, err error)
 
+	// GetBulkSecret retreaves all preconfigred secrets for this application.
```
Comment (yaron2) [inline client/client.go:60]: nit: retrieves

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: client/client.go [hunk 2]
```diff
@@ -57,6 +57,9 @@ type Client interface {
 	// GetSecret retreaves preconfigred secret from specified store using key.
```
Comment (yaron2) [inline client/client.go:57]: retrieves

File: client/client.go (hunk 1 above)
Comment (mchmarny) [reply client/client.go:60]: Hm, I ran make spell on this and for some reason didn't catch. On it.

File: client/client.go (hunk 2 above)
Comment (mchmarny) [reply client/client.go]: Fixed all 12 of those ;)

**Review by yaron2 — APPROVED** [review]
LGTM

**Pull request merged**